Reference-Platform/CECommon/OE.md

```diff
@@ -11,15 +12,12 @@ This wiki is not an introduction on OpenEmbedded or Yocto Project. If you are no
 
 In this wiki, we assume that the reader is familiar with basic concepts of OpenEmbedded.
 
-The support for DragonBoard 410c is available in the [meta-qcom BSP layer](http://git.yoctoproject.org/cgit/cgit.cgi/meta-qcom).
-
-The support for HiKey is available in the [meta-96boards BSP layer](https://github.com/96boards/meta-96boards).
-
-These layers have been tested with OpenEmbedded Core layer, and are expected to work with any other standard layers and of course any OpenEmbedded based distributions.
+The support for any board is available in the dedicated BSP Layers. These layers have been tested with OpenEmbedded Core layer, and are expected to work with any other standard layers and of course any OpenEmbedded based distributions.
```

Comment:
the sentence that really triggered my review is "It aims at being generic for any board support." . I don't want to say something like this. We can have a less generic introduction i believe.